braindecode/training/scoring.py

```diff
@@ -295,3 +297,47 @@ def on_epoch_end(self, net, dataset_train, dataset_valid, **kwargs):
                 cached_net, dataset_train, self.y_trues_
             )
         self._record_score(net.history, current_score)
+
+
+def predict_trials(module, dataset, return_targets=True):
```

Comment:
why adding 2 ways of doing the same thing ie the method and this public function?

Reply:
I guess `braindecode.training.scoring.predict_trials` exists because you don't need an EEGClassifier / EEGRegressor to make predictions. It is sufficient to have any kind of model given as PyTorch module as well as a braindecode dataset. 

However, you would expect your estimator to provide `predict..` right?